src/serac/physics/utilities/functional/detail/metaprogramming.hpp

```diff
@@ -118,3 +118,82 @@ inline constexpr void for_constexpr(lambda&& f)
 {
   detail::for_constexpr(f, std::make_integer_sequence<int, n>{}...);
 }
+
+// sam's templates for undoing for_constexpr for faster compile times
+template <int n1, typename lambda>
+void for_loop(const lambda& f)
```

Comment:
It's usually fine to pass these "by value" but if they're references they need to be forwarding/universal references, i.e., `lambda&& f`. I think `mutable` lambdas are not `const` and couldn't be used with a `const lambda& f`

Reply:
Alright I'll change it back to "pass by value".